### PR TITLE
Added definitions for pinch and cthulhu smileys

### LINKED ISSUE #236: Smileys and underscore
From zestedesavoir/zds-site#4958 :
> Le smiley >_< n'est pas converti en image sur la beta actuelle (beta de la v27).
> Le bug est présent sur les forums et les commentaires des tutos.
> 
> De plus, il y a un conflit entre l'underscore présents dans ledit smiley et ceux utilisés pour l'emphase.

Le bug n'est pas présent dans les tutos.

### PR DESCRIPTION
**Pull request by nils-van-zuijlen**
Closes #236

### CODE REVIEW

**Comment by vhf** [comment]
Don't worry about the tests, I'll fix the build.

File: packages/zmarkdown/config/remark.js [hunk 1]
```diff
@@ -144,6 +144,8 @@ const remarkConfig = {
       ':colere2:': '/static/smileys/mechant.png',
       ':ninja:': '/static/smileys/ninja.png',
       'x(': '/static/smileys/pinch.png',
+      '>_<': '/static/smileys/pinch.png',
+      '&lt;_&gt;': '/static/smileys/pinch.png',
```
Comment (vhf) [inline packages/zmarkdown/config/remark.js]: Do we need this?

File: packages/zmarkdown/config/remark.js [hunk 2]
```diff
@@ -144,6 +144,8 @@ const remarkConfig = {
       ':colere2:': '/static/smileys/mechant.png',
       ':ninja:': '/static/smileys/ninja.png',
       'x(': '/static/smileys/pinch.png',
+      '>_<': '/static/smileys/pinch.png',
```
Comment (vhf) [inline packages/zmarkdown/config/remark.js]: Could you please add a test to make sure this gets parsed properly?

File: packages/zmarkdown/config/remark.js (hunk 1 above)
Comment (nils-van-zuijlen) [reply packages/zmarkdown/config/remark.js]: As it is the current output of the editor, I thik it is useful.

File: packages/zmarkdown/__tests__/new-html-suite.test.js [hunk 3]
```diff
@@ -220,3 +220,17 @@ describe('ping', () => {
     remarkConfig.ping.pingUsername = () => false
   })
 })
+
+describe('smileys', () => {
+  it(`translates >_<`, () => {
+    const input = 'This is funny >_<'
+
+    return expect(await renderString(input)).toMatchSnapshot()
```
[the remark this answers is not in the document]
Comment (vhf) [reply packages/zmarkdown/__tests__/new-html-suite.test.js]: @nils-van-zuijlen go up ~10 lines to see how to check what a promise resolves to:

https://github.com/nils-van-zuijlen/zmarkdown/blob/dbf86e2c799ff0af0f6604619745fe3240de0ddf/packages/zmarkdown/__tests__/new-html-suite.test.js#L216

**Comment by nils-van-zuijlen** [comment]
Well. The smiley `^{;,;}^` is interpreted as `<sup>{;,;}</sup>`.
I don't know how to fix that. It might be a priority problem.

**Comment by vhf** [comment]
> I don't know how to fix that. It might be a priority problem.

I don't think it is. You seem to define a smiley and then test another one, could you please double-check what's the correct syntax for this smiley?

**Comment by vhf** [comment]
Rebased and close via 775ab6633bd61536f09cc7cc0f2ccc734526497c

Thank you @nils-van-zuijlen !

**Pull request closed without merging**